lib/core/config.js

```diff
@@ -242,13 +242,13 @@ Config.prototype.loadContractsConfigFile = function() {
     });
   }
   Object.keys(newContractsConfig.contracts).forEach(contractName => {
-      let contractGas = newContractsConfig.contracts[contractName].gas;
-      let contractGasPrice = newContractsConfig.contracts[contractName].gasPrice;
-      if (contractGas && contractGas != autoGas) {
-        newContractsConfig.contracts[contractName].gas = utils.getWeiBalanceFromString(contractGas, web3);
+      let gas = newContractsConfig.contracts[contractName].gas;
+      let gasPrice = newContractsConfig.contracts[contractName].gasPrice;
+      if (gas && gasPrice !== GAS_VALUE_AUTO) {
```

Comment:
I think this should be `gas && gas !== GAS_VALUE_AUTO`

Reply:
woops! sorry about that one!